packages/iTwinUI-react/src/core/Slider/Slider.tsx

```diff
@@ -437,6 +467,8 @@ export const Slider = React.forwardRef<HTMLDivElement, SliderProps>(
         ref={ref}
         className={cx(
           'iui-slider-component-container',
+          { 'iui-slider-horizontal': orientation === 'horizontal' }, // TODO: Is a ternary operator possible to select only horizontal or vertical?
+          { 'iui-slider-vertical': orientation === 'vertical' },
           { 'iui-disabled': disabled },
           className,
         )}
```

Comment:
Is there a way to use a ternary operator here? I was thinking, in case there is a typo and on both lines 470 and 471, we write `orientation === 'horizontal'`, then both `iui-slider-horizontal` and `iui-slider-vertical` classes will be included. In a ternary operator, this cannot happen, because it always only does either the true case or the false case.

Reply:
It is ok do as you have here since only can be horizontal or vertical. And Typescript wouldn't allow to type any other value here.

Reply:
**[DONE]** Thanks

Reply:
`` `iui-slider-${orientation}` ``

Using this line of code will work for slider orientation since orientation will never be undefined

Reply:
@adamhock Did you mean something like this? If so, I don't think it's working, unfortunately.
![image](https://user-images.githubusercontent.com/45748283/181645744-39f63918-c122-4d50-bb90-16169121e6ad.png)

Reply:
You shouldn't need the `: true,` portion of the code.

Reply:
**[DONE]** Ohh, I totally forgot that we can pass classes as plain text also! Yes, after removing it from the curly braces and removing the `: true`, it worked, thanks